### ISSUE TITLE
The mobile skip link to jump to the search results cannot be operated by assistive devices as the focusable element has been hidden.

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#3039]
The mobile skip link to jump to the search results cannot be operated by assistive devices as the focusable element has been hidden.

**Suggestion:**
This visible element is a <label> functioning as a bookmark link.
Probably adjusting the markup to the correct semantic element is the easiest fix. Exposing the input would be confusing, role-wise.

WCAG Guidelines:
https://www.w3.org/WAI/WCAG21/quickref/#bypass-blocks

**Comment by sandbergja** [comment]
This is actually an issue with the drupal website.  You can recreate by opening https://library.princeton.edu/find/all/plato in a mobile view.

The change will likely be [in this file](https://github.com/pulibrary/pul_library_drupal/blob/main/sites/all/modules/custom/all_search_page_configuration/all_search_page_configuration.pages_default.inc).
